CompressedFluids/data-updates.lua

```diff
@@ -1,2 +1,3 @@
 require "prototypes/fluid/compressor-fluid-updates"
 require "prototypes/recipe/compressor-fluid-updates"
+require "prototypes/mod-compatibility-final-updates"

```

Comment:
Shouldn't this be just updates instead of final updates?
```suggestion
require "prototypes/mod-compatibility-updates"
```

Reply:
Oops, my factorio instance was out of sync with the git checked out version. will do another round of testing and push fixes as needed